### ISSUE TITLE
Cannot paste multi-line block into multiple cursors

### ISSUE THREAD

**Issue by smlombardi** [Microsoft/vscode#30397]
- VSCode Version: Code - Insiders 1.14.0-insider (b8cd5b9556d8b70ea560d35b903422363f6c5c40, 2017-07-06T20:14:49.095Z)
- OS Version: Darwin x64 16.6.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|0.1.4
vscode-markdownlint|Dav|0.9.1
code-settings-sync|Sha|2.8.2
sort-lines|Tyr|1.3.0
Bookmarks|ale|0.15.2
project-manager|ale|0.18.1
path-intellisense|chr|1.4.2
vscode-eslint|dba|1.2.11
vscode-html-css|ecm|0.1.7
tslint|eg2|0.16.0
auto-rename-tag|for|0.0.14
Angular2|joh|2.3.3
ecdc|mit|0.10.3
HTMLHint|mka|0.3.3
vscode-autoprefixer|mrm|2.0.0
vscode-stylefmt|mrm|2.4.1
angular2-inline|nat|0.0.17
stylelint|shi|0.26.0
ayu|tea|0.4.0
change-case|wma|1.0.0

(4 theme extensions excluded)

---

Steps to Reproduce:

1. Let's say you have a multi-line block of html, such as
````
<div class="card-footer">
        <small>Optional Card Footer</small>
</div>
````
2. Copy it to clipboard
3. Make multiple cursors where you need it inserted. I needed it in 3 places.
4. Paste

You with up with _each line_ from the clipboard pasted into _each cursor_.

![jul-11-2017 11-50-03](https://user-images.githubusercontent.com/835148/28077217-2639d236-662f-11e7-9cda-dfc17ca44a76.gif)

**Comment by larsgk** [comment]
This is an important part of using multi cursor editing

**Comment by alexandrudima** [comment]
Fixed with the fix for #16155 (beb6a21322f46933b2c21b7f113067a4e715ace0)
